File: tests/fixtures/languages/sample.py
import os


class SampleService:
    def __init__(self, name: str) -> None:
        self.name = name

    def run(self) -> str:
        total = 0
        total += len(self.name)
        total += len(os.getcwd())
        total += 1
        return f"{self.name}:{total}"


def helper(value: int) -> int:
    result = value
    result += 1
    result += 2
    return result

File: tests/fixtures/languages/sample.rs
use std::fmt::Display;

pub struct SampleService {
    name: String,
}

impl SampleService {
    pub fn run<T: Display>(&self, value: T) -> String {
        let mut total = 0;
        total += self.name.len();
        total += 1;
        total += 2;
        total += 3;
        format!("{}:{value}:{total}", self.name)
    }
}

pub fn helper(value: i32) -> i32 {
    value + 1
}
